website/templates/landing.mako

```diff
@@ -336,11 +336,12 @@
 
     </div>
 
-
 </%def>
 
 <%def name="footer()">
-
+    <div style="margin-top:-25px">
```

Comment:
Instead of inline styling, you can override the margin for the `footer` class in landing-page.css. We do this for the homepage [here](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/static/css/pages/home-page.css#L50).